student.rb

```diff
@@ -1,10 +1,20 @@
 require_relative './person'
 class Student < Person
-  attr_accessor :classroom
+  attr_accessor :name
 
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
-    super(age, name, parent_permission)
+    super(age, name, parent_permission: parent_permission)
     @classroom = classroom
+    @classroom = nil
+  end
```

Comment:
- [ ] Please make sure the classroom variable is readable to work association as expected.